storage/innobase/xtrabackup/src/xbstream.c

```diff
@@ -246,6 +250,9 @@ get_one_option(int optid, const struct my_option *opt __attribute__((unused)),
 	case 'k':
 		hide_option(argument, &opt_encrypt_key);
 		break;
+  case 'V':
+		print_version();
+		exit(0);
```

Comment:
two comments
1. alignment of case
2. should it be lower case 'v'? (also does case matter?)

Reply:
1 . Done
2. lower case 'v' is already used for --verbose. So I've added upper case 'V' as the short variant of --version.